### DISCUSSION TITLE
About score of Assignment1,2

### DISCUSSION THREAD

**Discussion by hichoe95** [seominjoon/kaist-ai605#62] [Q&A] [answered]
I have a question about assignments score.

I was given only scores for each problem in assignments. 

Can I have the correct answers for assignments and the reason why I have those scores?

Regards,

**Comment by seominjoon** [comment, marked as answer]
I just uploaded the rubrics for A1 and A2. Please check KLMS.
If you have inquiries, please chat with the TA in charge or look out for office hour announcement during Finals week.
